packaging/docker/README.md

```diff
@@ -50,14 +50,33 @@ services:
       - /proc:/host/proc:ro
       - /sys:/host/sys:ro
       - /var/run/docker.sock:/var/run/docker.sock:ro
+      - /path/to/actual/docker/on/the/host:/usr/bin/docker
 ```
 
 ### Docker container names resolution
 
-If you want to have your container names resolved by netdata it needs to have access to docker group. To achive that just add environment variable `PGID=999` to netdata container, where `999` is a docker group id from your host. This number can be found by running:
-```bash
-grep docker /etc/group | cut -d ':' -f 3
-```
+If you want to have your container names resolved by netdata, you need to do two things:
+1) Make netdata user be part of the group that owns the socket.
+   To achive that just add environment variable `PGID=999` to netdata container,
+   where `999` is practically the group id of the group assigned to the docker socket, on your host.
+   This group number can be found by running the following (if socket group ownership is docker):
+   ```bash
+   grep docker /etc/group | cut -d ':' -f 3
```

Comment:
So why do you mention 999, if I need to find it? It's misleading. It's either 999 and I don't need the command, or it's PGID=[GROUP_NUMBER] and we say you can find  [GROUP_NUMBER]  by running...